docs/components/plugins/AuthSessionElements.tsx

```diff
@@ -25,18 +27,17 @@ const STYLES_LINK = css`
 `;
 
 const STYLES_BUTTON = css`
-  display: inline-flex;
+  margin-bottom: ${spacing[4]}px;
 `;
 
 export const CreateAppButton: React.FC<{ href: string; name: string }> = ({ href, name }) => (
-  <a
+  <Button
     css={STYLES_BUTTON}
-    className="snack-inline-example-button"
     href={href}
     target="_blank"
-    rel="noreferrer">
+    iconRight={<ArrowUpRightIcon color={theme.palette.white} />}>
```

Comment:
```suggestion
    iconRight={<ArrowUpRightIcon color={theme.button.primary.foreground} />}>
```

This does not make a visual change, but I prefer to use the button colors when adding elements to buttons (in the case we ever change our button styles).

Reply:
Nice catch, I have forgot about those theme variables, thanks Jon! 🙏 

I will update the places where I have used `palette.white` before, in the separate PR.